src/__tests__/filter-style-values.test.ts

```diff
@@ -23,7 +23,7 @@ describe("filterStyleValues", () => {
     expect(result).toEqual(expected);
   });
 
-  it("prevents @import statements", () => {
+  it("prevents @ rule statements", () => {
```

Comment:
[nitpick] The official name for these things is literally [at-rules](https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Syntax/At-rules) (how creative). Could we use that here?

```suggestion
  it("prevents at-rules", () => {
```